### PR TITLE
⚠️ Add migration and deprecation notice

### PR DESCRIPTION
**Pull request by cairoeth**

### CODE REVIEW

File: docs/modules/ROOT/pages/faq.adoc [hunk 1]
```diff
@@ -1,72 +1,44 @@
...
-Yes. Our team will continue to maintain Defender for the foreseeable future. We will communicate our plan as things progress, ensuring there is an upgrade path for existing Defender users.
-
-=== Will I be able to import my existing Defender configuration to Defender 2.0 beta?
+== Is the existing Defender version still supported?
 
-Yes. All existing configurations for Autotask, Sentinels, etc., will be automatically imported to Defender 2.0 beta under the new features naming convention.
+Defender 1.0 is now deprecated in favor of Defender 2.0. On May 21st 2024, users will no longer have access to Defender 1.0 UI and API. Users that haven't migrated will be forced to migrate. Learn how to migrate xref:migration.adoc[here].
 
-=== Is there any timeline for Defender 2.0 to be generally available?
+== Once I migraete to Defender 2.0, can I continue using Defende 1.0?
```
Comment (shahnami) [inline docs/modules/ROOT/pages/faq.adoc:18]:
```suggestion
== Once I migrate to Defender 2.0, can I continue using Defender 1.0?
```

File: docs/modules/ROOT/pages/faq.adoc [hunk 2]
```diff
@@ -1,72 +1,44 @@
...
+We offer a Service Level and Support Agreement (SLA) for paid subscriptions. Learn more xref:index.adoc#sla[here].
 
-=== Does Defender 2.0 beta offer the same level of support?
+== How can I get an upgrade my tenant account?
 
-Yes. You can reach out and submit a support request for Defender 2.0 in the same channels as the existing Defender version. Additionally, you can submit feedback or report an issue directly from the web application using the dedicated feedback form.
+You can use the billing page to upgrade your tenant account to a higher tier https://defender.openzeppelin.com/v2/#/billing/[here, window=_blank].
 
-=== How can I get an additional quota or upgrade my tenant account?
+== Where can I see my tier quota usage?
```
Comment (shahnami) [inline docs/modules/ROOT/pages/faq.adoc:42]: Nit: we use "tier" and "subscription" interchangeably. Should we stick to one?

**Review by shahnami — APPROVED** [review]
LGTM! Thanks :)

File: docs/modules/ROOT/pages/faq.adoc (hunk 2 above)
Comment (cairoeth) [reply docs/modules/ROOT/pages/faq.adoc:42]: i see that we are using `subscription` to refer to general paid tiers and `tier` for a specific one, so should be good for now

**Pull request merged**